package.json

```diff
@@ -24,7 +24,19 @@
     "nwb-sass": "^0.10.2",
     "typescript": "^3.9.5"
   },
+  "files": [
+    "./src/**/*"
+  ],
+  "bugs": {
+    "url": "https://github.com/cjquines/qboard/issues"
+  },
   "author": "cjquines",
+  "contributors": [
+    {
+      "name": "Avi Mehra",
```

Comment:
lol not as if we are ever going to publish this on npm or something

Reply:
Potential SEO lol

Reply:
lol fine